deltachat-ffi/deltachat.h

```diff
@@ -2588,6 +2594,85 @@ char* dc_get_last_error (dc_context_t* context);
 void dc_str_unref (char* str);
 
 
+/**
+ * @class dc_backup_sender_t
+ *
+ * Set up another device.
+ */
+
+/**
+ * Create an object for sending a backup to another device.
+ * The backup is sent through an peer-to-peer channel which is bootstrapped by a QR-code.
+ * As a backup contains the whole state of the account, including credentials,
+ * this can be used to setup a new device.
+ *
+ * @memberof dc_backup_sender_t
+ * @param context The context.
+ * @param folder Path to create the backup in before it is sent out.
+ *     The path must exist and being accessible at least until the other device has received all data.
+ *     The folder is not cleaned up by the backup send;
+ *     a temporary directory seems to be good place therefore.
```

Comment:
I'm not a UI developer, but this seems a bit awkward and something that the core could all handle internally?

Reply:
I don't have strong opinions in either direction @r10s what do you think?

Reply:
in general, it is nicer if the UI would not need to take care about such details.

but not sure how easy it really is for the core to really fulfill the requirements, so, i am also fine with leave things up to the UIs for now, where we can more safely pass a directly that is really handled by the OS.